example/src/main/java/com/klarna/example/MainActivity.java

```diff
@@ -132,6 +133,7 @@ public void onBuyPressed(View view) {
         } else {
             // Open Klarna registration
             Intent intent = new Intent(this, RegistrationActivity.class);
+            intent.putExtra(RegistrationActivity.EXTRA_SETTINGS, new RegistrationSettingsBuilder().setConfirmedUserDataId("abcd").build());
```

Comment:
I don't think it's a good idea to add this here, as it's suppose to be simple example and not include "advanced" feature
Also what is abdc?